packages/table/src/table.tsx

```diff
@@ -1004,6 +1092,53 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.handleSelection([]);
     }
 
+    private handleFocusMove = (e: KeyboardEvent, direction: "up" | "down" | "left" | "right") => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const {focusedCellCoordinates} = this.state;
+        const newFocusedCellCoordinates = { col: focusedCellCoordinates.col, row: focusedCellCoordinates.row };
+        const { grid } = this;
+        if (direction === "up") {
+            newFocusedCellCoordinates.row--;
```

Comment:
meh, it's not like the single-instruction inc/dec is ambiguous or difficult to understand. it's only during assignment like `tmp = other++;` when things go wrong.